readme.md

```diff
@@ -11,37 +16,79 @@
 - Push changes to GitHub
 - Submit a pull request
 
-> We will review your changes, if we like them we will add them.
+> A team member will review your changes, please keep an eye out for any requested changes during this review.  Once all the code is working and they like the changes, they will merge it into the repo.  Thank you for the help!
+
+[Back To Top](#Code-experiment-meetup)
+
+
+# Git Workflow For Users On Contributors List
+
+> Please follow these steps each day you sit down to code.
+
+### Find Something To Do
+
+- Head over to the Glo board and find something off the TODO section.
+- Head over to the Issues section.
 
-## Git Workflow
+### You Found Something To Do
 
-- Please follow these steps each day you sit down to code
+- Let the team know what you decided to work on.
+- Assign yourself to that task.
+- Drag that task to In Progress.
 
-### Pull Latest Code
+### Pull The Latest Code
 
+- Make sure you don't have any random changes and pull the latest code.
 ```
-$ git checkout master
+$ git status
+$ git checkout development
 $ git pull
 ```
 
-### Create your branch
+### Create your feature branch
 
-- Please create a new branch for each day of updates
+- Please create a new branch for your new task.
 
 ```
-$ git checkout -b "yourName-date"
+$ git checkout -b card-name
+OR
+$ git checkout -b issue#number
 ```
+> Examples:
+```
+$ git checkout -b COD-10
+$ git checkout -b issue#5
+```
+
+### CODE!
+
+- Your free to code.  We appreciate your help.
+- Don't be scared to commit as much as you want during your feature branch.
 
-### Finished with your new changes do this
+### You Finished your new changes
 
+- Make sure you have working code!
+- Check to make sure you don't have any last minute commits and then push it to github.
 ```
-$ git add .
-$ git commit -m "you commit message"
-$ git push
+$ git status
+$ git push origin branch-name
 ```
 
-> Communicate with another team member that you have pushed a new branch
+> Example Push.
+```
+$ git status
+$ git push origin COD-10
+```
 
 ### Pull Request
 
-- Hop onto github and work through the pull request together as a team
\ No newline at end of file
+- Hop onto github and create a pull request.
+- Attach pull request to the Glo board card.
```

Comment:
Add compare and pull request. On the open pull request page, change the title to include your branch name in brackets. Add any extra comments in the description field of pull request. Then do the pull request.